CHANGELOG.rst

```diff
@@ -13,6 +13,9 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+* Add linter for missing feature toggle annotations ("toggle-missing-annotation").
```

Comment:
1. See https://github.com/edx/edx-lint/pull/147/files for a potential adr and how-to for adding new checks.  Let me know what you think.
2. Please add a note to see https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html for fixing violations.

Reply:
> See https://github.com/edx/edx-lint/pull/147/files for a potential adr and how-to for adding new checks. Let me know what you think.

:+1: I commented there.

> Please add a note to see https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html for fixing violations.

Do you mean adding information to the pylint check description messages? Should we add it to every message? Isn't this overkill?

Reply:
I meant to the changelog message. Not sure where the best place or places would be to help people make the connection, and maybe you’ll have other ideas, but the changelog seemed reasonable.